### ISSUE TITLE
VTOL/WiGE destruction and infantry

### ISSUE THREAD

**Issue by AlphaSled** [MegaMek/megamek#2034]
v.47.7

Relevant rules are on p. 224 of Total Warfare, under the VTOL/WiGE Carriers subheader.

As per the Total Warfare rules, infantry being transported by VTOLs or WiGEs can be destroyed in one of three ways:
a) The VTOL/WiGE is destroyed in the air, at which point the infantry is automatically destroyed with it.
b) The VTOL/WiGE is destroyed in a crash landing, at which point the infantry is automatically destroyed with it.
c) The VTOL/WiGE survives a crash landing, at which point each infantry squad or platoon needs to roll to survive (1-3 for survival, 4-6 for destruction).

From what I've seen in my testing, Megamek is always allowing infantry to survive no matter what the scenario.
Attached are a few save files where I tested the different scenarios.

[vtol (crash survived)t.sav.gz](https://github.com/MegaMek/megamek/files/4842719/vtol.crash.survived.t.sav.gz)
[vtol (destruction in air).sav.gz](https://github.com/MegaMek/megamek/files/4842720/vtol.destruction.in.air.sav.gz)
[vtol (crash destruction).sav.gz](https://github.com/MegaMek/megamek/files/4842721/vtol.crash.destruction.sav.gz)
[wige (crash destruction).sav.gz](https://github.com/MegaMek/megamek/files/4842722/wige.crash.destruction.sav.gz)

**Comment by Scoppio** [comment]
@Sleet01 I believe you fixed this correct?

**Comment by Sleet01** [comment]
@Scoppio yes, this was fixed around 0.50.05, with the various infantry and cargo bay fixes.
(Apologies, I missed this while I was on vacation)
